src/app/rpmostree-builtin-kargs.cxx

```diff
@@ -44,6 +46,8 @@ static GOptionEntry option_entries[] = {
   { "append", 0, 0, G_OPTION_ARG_STRING_ARRAY, &opt_kernel_append_strings, "Append kernel argument; useful with e.g. console= that can be used multiple times. empty value for an argument is allowed", "KEY=VALUE" },
   { "replace", 0, 0, G_OPTION_ARG_STRING_ARRAY, &opt_kernel_replace_strings, "Replace existing kernel argument, the user is also able to replace an argument with KEY=VALUE if only one value exist for that argument ", "KEY=VALUE=NEWVALUE" },
   { "delete", 0, 0, G_OPTION_ARG_STRING_ARRAY, &opt_kernel_delete_strings, "Delete a specific kernel argument key/val pair or an entire argument with a single key/value pair", "KEY=VALUE"},
+  { "append-if-missing", 0, 0, G_OPTION_ARG_STRING_ARRAY, &opt_kernel_append_if_missing_strings, "If not already present, append kernel argument; useful with e.g. console= that can be used multiple times. empty value for an argument is allowed", "KEY=VALUE" },
+  +  { "delete-if-present", 0, 0, G_OPTION_ARG_STRING_ARRAY, &opt_kernel_delete_if_present_strings, "If present, delete a specific kernel argument key/val pair or an entire argument with a single key/value pair", "KEY=VALUE"},
```

Comment:
I wonder if for these we should just simplify it to e.g. "Like --append, but does nothing if the key is already present". WDYT?

Reply:
I think that would make a description depend on the other one, so if they are not both visualized at the same time, the new one would seem incomplete. On the other hand, if  the user see both descriptions together is probably friendlier and better to simplify as you say. But I don't know how it would appear being used.